packages/plans-grid-next/src/hooks/data-store/use-grid-plans-for-comparison-grid.ts

```diff
@@ -0,0 +1,52 @@
+import {
+	FeatureList,
+	PLAN_ENTERPRISE_GRID_WPCOM,
+	PLAN_HOSTING_TRIAL_MONTHLY,
+} from '@automattic/calypso-products';
+import { useMemo } from 'react';
```

Comment:
This too.

Reply:
Updated: [c41c49f](https://github.com/Automattic/wp-calypso/pull/87377/commits/c41c49f4f2be4261a6f3afe5fdc852277d1d5655)